docs/config/README.md

```diff
@@ -296,15 +300,16 @@ date is read from the `AWSUME_EXPIRATION` env var.
-| `format`            | `'on [$symbol($profile )(\($region\) )(\[$duration\])]($style)'` | The format for the module.                                        |
-| `symbol`            | `"☁️ "`                                                          | The symbol used before displaying the current AWS profile.        |
-| `region_aliases`    |                                                                  | Table of region aliases to display in addition to the AWS name.   |
-| `profile_aliases`   |                                                                  | Table of profile aliases to display in addition to the AWS name.  |
-| `style`             | `"bold yellow"`                                                  | The style for the module.                                         |
-| `expiration_symbol` | `X`                                                              | The symbol displayed when the temporary credentials have expired. |
-| `disabled`          | `false`                                                          | Disables the `AWS` module.                                        |
+| Option                | Default                                                          | Description                                                                          |
+| --------------------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
+| `format`              | `'on [$symbol($profile )(\($region\) )(\[$duration\])]($style)'` | The format for the module.                                                           |
+| `symbol`              | `"☁️ "`                                                          | The symbol used before displaying the current AWS profile.                           |
+| `region_aliases`      |                                                                  | Table of region aliases to display in addition to the AWS name.                      |
+| `profile_aliases`     |                                                                  | Table of profile aliases to display in addition to the AWS name.                     |
+| `style`               | `"bold yellow"`                                                  | The style for the module.                                                            |
+| `expiration_symbol`   | `X`                                                              | The symbol displayed when the temporary credentials have expired.                    |
+| `disabled`            | `false`                                                          | Disables the `AWS` module.                                                           |
+| `display_empty_creds` | `false`                                                          | If true displays info even if credentials or credential_process have not been setup. |
```

Comment:
Please mention SSO in this description or make it more generic. I'm also not sure if "empty" is the appropriate word here. How about "missing", "without" or "no"?

Reply:
I went for "force_display" as option name. So it can be used in the future for anything else even not related to creds.
What do you think?

Reply:
I went for "force_display" as option name. So it can be used in the future for anything else even not related to creds.
What do you think?

Reply:
It's fine.